### DISCUSSION TITLE
Query About Re-initiating Payment for Failed Reservation Transactions

### DISCUSSION THREAD

**Discussion by TommsNL** [liedekef/events-made-easy#502] [Q&A] [answered]
Hi All/Franky,

Is there a standard approach or pattern in dealing with failed transactions, specifically in re-allowing users to attempt payment again? Or should this be done via the payment gateway and setting the reservation to paid manually?

Greets

**Comment by liedekef** [comment]
If you send them the payment url (#_PAYMENT_URL), they can click on it again until it gets paid for :-)

**Comment by TommsNL** [comment]
Wow you are fast as lightning! So I could add #_PAYMENT_URL to a standard e-mail for example and send them e-mail via the EME system?

**Reply by liedekef** [reply]
Only when you add it to a booking-related email (either when they book for an event or via the EME mail system, tab 'event related emails' and as type 'booking mails').

**Reply by TommsNL** [reply]
Would the "re-send e-mail for bookings"  (Stuur nogmaals de e-mail voor goed te keuren boekingen). Also work?

**Reply by TommsNL** [reply]
Or would the " Verstuur boekingsgerelateerde e-mail naar geselecteerde boekingen"  and a custom e-mail with the #_PAYMENT_URL be a better option?

**Reply by liedekef** [reply, marked as answer]
Both will work (the "resend email for bookings" will of course only work if you change the used format to include #_PAYMENT_URL )

**Reply by TommsNL** [reply]
Thanks! This works! 

One last related question..

Would the conditional check like:
[eme_if tag='#ESC_IS_PAID' eq='1'] work in these mails?

**Reply by liedekef** [reply]
it should, but you can just also select the bookings not paid yet, or set the option "Only send mails to attendees who did not pay yet":

![image](https://github.com/liedekef/events-made-easy/assets/75667/33ef3025-c503-426d-b9d3-6f9005c516f4)

**Reply by TommsNL** [reply]
Thanks for the quick replies.
